apps/web/src/pages/user-management/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { useEffect, useState } from 'react';
+
+import { UserManagementTable } from 'src/components/display/UserManagementTable';
+import withAuth from 'src/components/Keycloak';
+import { PageOptions, Pagination } from 'src/components/Pagination';
+import { getEmployees } from 'src/services/user-management';
+import { ValidRoles } from '@services';
+import { EmployeeRO } from '@ien/common';
+import { Spinner } from 'src/components/Spinner';
+import { EmployeeFilters } from 'src/components/user-management/UserFilter';
+interface SearchOptions {
+  name?: string;
+  role?: string[];
+  status?: string;
+  sortKey?: string;
+  order?: 'ASC' | 'DESC';
+  limit?: number;
+  skip?: number;
+}
+
+export interface EmployeeFilterOptions {
+  name?: string[];
+  role?: string[];
+  skip?: number;
+  limit?: number;
+}
+
+const DEFAULT_PAGE_SIZE = 10;
+
+const UserManagement = () => {
+  const [employees, setEmployees] = useState<EmployeeRO[]>([]);
+  const [loading, setLoading] = useState(false);
+
+  // will change this when search functionality is figured out
+  const name = '';
+
+  const [filters, setFilters] = useState<Partial<EmployeeFilterOptions>>({});
+  const [role, setRole] = useState<string[] | undefined>([]);
+  const [sortKey, setSortKey] = useState('');
+  const [order, setOrder] = useState<'ASC' | 'DESC'>('DESC');
+  const [limit, setLimit] = useState(DEFAULT_PAGE_SIZE);
+  const [page, setPage] = useState(1);
+  const [total, setTotal] = useState(0);
+
+  const searchEmployees = async (options: SearchOptions) => {
```

Comment:
I noticed the same pattern in applicants page but it seems unnecessary just to wrap a function without additional processes.

Reply:
agreed, will refactor